octoprint/tests/test_octoprint.py

```diff
@@ -0,0 +1,55 @@
+import mock
+import pytest
+
+from datadog_checks.base import ConfigurationError
+from datadog_checks.base.stubs.aggregator import AggregatorStub
+
+MOCK_RESPONSE = {
+    "rpi_core_temp": 0.0,
+    "printer_state": 1,
+    "pct_completed": 0,
+    "print_job_time": 1,
+    "print_job_time_left": 9999999,
+    "current_tool_temp": 50.0,
+    "target_tool_temp": 190.0,
+    "current_bed_temp": 68.0,
+    "target_bed_temp": 70
+}
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+@mock.patch('datadog_checks.octoprint.OctoPrintCheck._call_command')
```

Comment:
```suggestion
@mock.patch('datadog_checks.octoprint.OctoPrintCheck.get_api_info')
```
Your check doesn't have a `_call_command` method.